src/contractor/contractor.cpp

```diff
@@ -756,7 +768,8 @@ EdgeID Contractor::LoadEdgeExpandedGraph(
         std::ofstream geometry_stream(geometry_filename, std::ios::binary);
         if (!geometry_stream)
         {
-            throw util::exception("Failed to open " + geometry_filename + " for writing");
+            const std::string message{"Failed to open " + geometry_filename + " for writing"};
+            throw util::exception(std::move(message));
```

Comment:
Huh, why does this even compile? You shouldn't be able to move from a const object.